roles/system_packages/defaults/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
 # number of times package install task should be retried
-pkg_install_retries: 4
+pkg_ops_retries: 4
```

Comment:
IMO it's acceptable to have a less precise semantic here in order to not break compat.
Possible alternative: create the new variable using the value of the old as a default, + add a comment and a note in the release-note to deprecate it. But really it's not a big deal to keep the name as is IMO.